_documentation/user/flags/tiny.md

```diff
@@ -5,4 +5,16 @@ type: Good
 summary: Tank is small and can get through small openings. Very hard to hit.
 ---
 
-...
+The Tiny (T) flag allows the player to be smaller in tank size than normal, all while retaining all other properties, such as the tank speed, shot speed, or jump height.
+
+It is configured by the `_tinyFactor` server setting, which is multiplied by the normal tank size to get shrunken `_tankLength` and `_tankWidth` values for the tank holding the flag.
```

Comment:
this is good information, but is it more appropriate in the Server Settings section?

Reply:
@jwmelto thanks for the feedback; so far, we have only used the Server Settings section to *list* the server settings that directly affect the flag, everything else was explained or expanded in either Summary or Tactics sections. We can change that, but then we will have to reflect it in all other closed cases as well.

Reply:
I think this is something we can definitely revisit once we start officially moving over BZDB variable documentation to the new site.

Since this has started to grow stale, I'll merge this as-is.